examples/burial-form/ReviewPage.jsx

```diff
@@ -0,0 +1,345 @@
+import React from 'react';
+import {
+  DebuggerView
+} from '@department-of-veterans-affairs/va-forms-system-core';
+import { useFormikContext } from 'formik';
+import { Page } from '@department-of-veterans-affairs/va-forms-system-core';
+import { Link } from 'react-router-dom';
+import { VaOnThisPage } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+const transformFieldValue = (key, field) => {
+  if ( field.value === 'true' | field.value === true ) {
+    return 'Yes';
+  }
+  if (field.value === 'false' | field.value === false ) {
+    return 'No';
+  }
+  if (["from", "to", "veteranDateOfBirth", "deathDate", "burialDate"].indexOf(key) > -1) {
+    const date = new Date(field.value);
+    return date.toLocaleDateString("en-US", { day: 'numeric', month: 'long', year: 'numeric' });
+  }
+  if (["amountIncurred", "amountGovtContribution"].indexOf(key) > -1) {
+    return "$" + field.value
+  }
+  return field.value;
+}
+
+const bufferFields = (fields, rank = 0) => {
+  let buffer = [];
+  for (const [key, field] of Object.entries(fields)) { 
+    buffer.push(recurseField(key, field, rank));
+  }
+  return buffer;
+}
+
+const recurseField = (key, field, rank = 0) => {
+  if (field.value === "" | field.value === 0 | field.value === null | field.value === undefined) return;
+  const fieldLabel = field.label && <label className='vads-u-margin-top--1 review-page--page-info--label-text'>{field.label}:</label>
+
+  if ((typeof field.value) === 'object') {
+    return (<div className={`level-${rank}-field-${key}`} key={`level-${rank}-field-${key}`}> {bufferFields(field.value, rank+1)}</div>)
+  }
+  else {
+    return (
+      <div className={`level-${rank}-field-${key}`} key={`level-${rank}-field-${key}`}> 
+        {fieldLabel} 
+        <span className={'review-page--page-info--value-text field-value' + (rank > 0 && ` field-value-level-${rank}`)}> { transformFieldValue(key, field) }</span>
+      </div>
+    );
+  }
+}
+
+export default function ReviewPage(props) {
```

Comment:
Looks like props isn't being used here, is the `title` prop suppose to be used somewhere?